cabal-install/Distribution/Client/SetupWrapper.hs

```diff
@@ -365,7 +365,7 @@ runSetup verbosity setup args0 = do
 -- verbosity applies to ALL commands.
 verbosityHack :: Version -> [String] -> [String]
 verbosityHack ver args0
-    | ver >= mkVersion [1,25] = args0
+    | ver >= mkVersion [2,1]  = args0
```

Comment:
FWICT, every time we use `filterConfigureFlags`, we pass the result to the setup wrapper, so maybe the code in eb0593ed7690e402c288011dc77964e404bb78d9 is not strictly needed.